### PR TITLE
Update README with latest changes

### PR DESCRIPTION
**Pull request by notFloran**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ You have to activate the bundle:
 ```php
 // in app/AppKernel.php
 public function registerBundles() {
```
Comment (nclsHart) [inline README.md]: According to the [documentation](https://symfony.com/doc/current/bundles/best_practices.html#installation-instructions), it's a best practice to describe instructions for both flex and legacy ways in README.

Comment (notFloran) [reply README.md]: Ah nice catch, I didn't know this page

File: README.md [hunk 2]
```diff
@@ -64,31 +92,32 @@ class AppSchema implements SchemaInterface
...
 ```yml
-app.settings_schema:
-    class: Hexanet\Si\AppBundle\Settings\AppSchema
+App\Settings\AppSchema:
     tags: [hexanet.settings_schema]
 ```
 
+> The bundle provide autoconfiguration for class that implement `SchemaInterface`.
+
 After that we can use the `sf hexanet:settings:setup` command to generate all the settings, if a setting already exists the command ignores it.
```
Comment (nclsHart) [inline README.md]: Maybe `php bin/console` would be more standard than `sf`.

**Review by nclsHart — CHANGES_REQUESTED** [review]

**Review by nclsHart — APPROVED** [review]

**Comment by notFloran** [comment]
fix #6

**Pull request merged**